test/dataCollectionAppTest.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+require_once ('../getFunctions.php');
+
+use PHPUnit\Framework\TestCase;
+
+class dataCollectionAppTest extends TestCase {
+
+    public function testgetNameImg() {
+        $expected = "<img alt=\"Circle\"src=\"Url\" />";
+        $actual = getNameImg(["ShapeName" => "Circle", "imgurl" => "Url"]);
+
+        $this->assertSame($expected, $actual);
+    }
+
+    public function testErrorWhenString() {
```

Comment:
you need to specify the function you are testing in the name, try `testgetNameImg_error_string`